src/drivers/intel/cavs/alh.c

```diff
@@ -63,14 +64,15 @@ static int alh_remove(struct dai *dai)
 
 static int alh_get_handshake(struct dai *dai, int direction, int stream_id)
 {
-	/* TODO */
-	return 0;
+	return alh_handshake_map[stream_id];
```

Comment:
Is stream_id passed in from host ? Should we validate it's value before any array lookup ?